src/Fmt_odoc.mli

```diff
@@ -9,7 +9,11 @@
  *                                                                    *
  **********************************************************************)
 
-val fmt : Octavius.Types.t -> Fmt.t
+val init :
+  (Source.t -> Cmts.t -> Conf.t -> Parsetree.structure -> Fmt.t) -> unit
+(** Initialize internal state *)
+
+val fmt : Conf.t -> Octavius.Types.t -> Fmt.t
```

Comment:
Instead of passing `Conf.t` as argument and `format` as global state, I think it would be better to create a new type `type conf = { fmt_code : string -> Fmt.t }` that can be extended in the future and abstract a lot of things from this module (`Source`, `Cmts`. `Parsetree`, `Conf`).

Reply:
Done in 3fe083f are you really sure it's better? if yes I will do the same for the ref in Normalize.ml

Reply:
Yes it's definitely better. I had in mind a simpler `string -> Fmt.t` that abstract modules that `Fmt_odoc` don't need (`Source`, `Cmts`, `Conf`, the parsetree and the parsing code)

Reply:
done